core/src/main/java/de/danoeh/antennapod/core/syndication/namespace/PodcastIndex.java

```diff
@@ -0,0 +1,40 @@
+package de.danoeh.antennapod.core.syndication.namespace;
+
+import org.jsoup.helper.StringUtil;
+import org.xml.sax.Attributes;
+import de.danoeh.antennapod.core.feed.FeedFunding;
+import de.danoeh.antennapod.core.syndication.handler.HandlerState;
+
+public class PodcastIndex extends Namespace {
+
+    public static final String NSTAG = "podcast";
+    public static final String NSURI = "https://github.com/Podcastindex-org/podcast-namespace/blob/main/docs/1.0.md";
+    public static final String NSURI2 = "https://podcastindex.org/namespace/1.0";
+    private static final String URL = "url";
+    private static final String FUNDING = "funding";
+    private FeedFunding funding;
+
+    @Override
+    public SyndElement handleElementStart(String localName, HandlerState state,
+                                          Attributes attributes) {
+        if (FUNDING.equals(localName)) {
+            String href = attributes.getValue(URL);
+            if (funding == null) {
```

Comment:
Why this null check? Wouldn't that mean that multiple funding tags write to the same instance of the class?

Reply:
`funding` is never initialize, I only initialize it if there is a `funding` tag

Reply:
If there is a second `funding` tag, it is not initialized again, if I read this correctly. That would mean that the second tag uses the same object instance as the first tag and therefore overwrites it.